developer-hub/src/tutorial/currency_simple/mod.rs

```diff
@@ -0,0 +1,733 @@
+//! # Currency Pallet
+//!
+//! By the end of this tutorial, you will write a small FRAME pallet (see
+//! [`crate::polkadot_sdk::frame_runtime`]) that is capable of handling a simple crypto-currency.
+//! This pallet will:
+//!
+//! 1. Allow a anyone to mint new tokens into accounts (which is obviously not a great idea for a
+//!    real system).
+//! 2. Allow any user that owns tokens to transfer them to others.
+//! 3. Tracks of the total issuance of all tokens at all times.
+//!
+//! > This tutorial will build a currency pallet from scratch using only the lowest primitives of
+//! > FRAME, and is mainly intended for education, not *applicability*. For example, almost all
+//! > FRAME-based runtimes use various techniques to re-use a currency pallet instead of writing
+//! > one. Further advance FRAME related topics are discussed in [`crate::reference_docs`].
+//!
+//! ## Topics Covered
+//!
+//! The following FRAME topics are covered in this tutorial. See the rust-doc of the associated
+//! items to know more.
+//!
+//! - [Storage](frame::pallet_macros::storage`)
+//! - [Call](frame::pallet_macros::call)
+//! - [Event](frame::pallet_macros::event)
+//! - [Error](frame::pallet_macros::error)
+//! - Basics of testing a pallet.
```

Comment:
Do we want to link this?
```suggestion
//! - Basics of testing a pallet
```